frontend/locales/en/strings.json

```diff
@@ -5,11 +5,61 @@
     }
   },
   "wizard": {
-     "headNode": {
-       "memoryBasedSchedulingEnabled": {
-         "label": "Slurm Memory Based Scheduling Enabled",
-         "help": "If enabled, users may use <i>--mem</i> to specify the amount of memory per node required by a job. For more information, see the <0>Slurm documentation for ConstrainRAMSpace</0>"
-       }
+    "headNode": {
+      "memoryBasedSchedulingEnabled": {
+        "label": "Slurm Memory Based Scheduling Enabled",
+        "help": "If enabled, users may use <i>--mem</i> to specify the amount of memory per node required by a job. For more information, see the <0>Slurm documentation for ConstrainRAMSpace</0>"
+     }
+   },
+    "queues": {
+      "schedulableMemory": {
+        "name": "Schedulable Memory (MiB)",
+        "description": "Amount of memory in MiB to be made available to jobs on the compute nodes of the compute resource",
+        "placeholder": "Leave empty for default",
+        "help": "The default value is 95 % of the memory advertised by EC2."
+      }
+   },
+    "multiuser": {
+      "domainName": {
+        "name": "Domain Name*",
+        "description": "The Active Directory (AD) domain that you use for identity information.",
+        "help": "This property corresponds to the sssd-ldap parameter that's called ldap_search_base."
+      },
+      "domainAddress": {
+        "name": "Domain Address*",
+        "description": "The URI or URIs that point to the AD domain controller that's used as the LDAP server.",
+        "help": "The URI corresponds to the sssd-ldap parameter that's called ldap_uri. The value can be a comma separated string of URIs. To use LDAP, you must add ldap:// to the beginning of the each URI."
+      },
+      "passwordSecretArn" : {
+        "name": "Password Secret ARN*",
+        "description": "The URI or URIs that point to the AD domain controller that's used as the LDAP server.",
```

Comment:
The description and help here are wrong because of a copy-paste error, will fix in a subsequent CR